app/src/main/kotlin/compiler/ast/AstNode.kt

```diff
@@ -0,0 +1,5 @@
+package compiler.ast
+
+abstract class AstNode {
```

Comment:
Maybe it would be more consistent with the rest of the code if `AstNode` was an interface.

Reply:
Ok